### ISSUE TITLE
Client terminates on corrupted network weights

### ISSUE THREAD

**Issue by sergiovieri** [LeelaChessZero/lczero-client#41]
```
...
option name Ignore alternatives to checkmate type check default false
option name Configuration file path type string default lc0.config
option name Do debug logging into file type string default 
uciok
Loading weights file from: networks/1b81faa3ebe18a6b93eb5a046e521d087fa30337a85aa9acc35114e6726b5ed8
error Invalid weight file: parse error.
2018/08/24 01:42:39 lc0_main.go:403: Bestmove has timed out, aborting match
2018/08/24 01:42:39 lc0_main.go:336: Waiting for candidate to exit.
2018/08/24 01:42:39 lc0_main.go:326: Waiting for baseline to exit.
2018/08/24 01:42:39 lc0_main.go:552: playMatch: timeout
```

Client seems to terminate when network weights are corrupted. It should redownload weights after playMatch timeout.

**Comment by sergiovieri** [comment]
Isn't after lc0_main.go:552 it should go to lc0_main.go:661 and sleeps for 30 seconds, but it was not printed on the log.

**Comment by sergiovieri** [comment]
Fixed with #44
